src/data/api.js

```diff
@@ -5,14 +5,26 @@ import { ExamAction } from '../constants';
 const BASE_API_URL = '/api/edx_proctoring/v1/proctored_exam/attempt';
 
 export async function fetchExamAttemptsData(courseId, sequenceId) {
-  const url = new URL(
-    `${getConfig().LMS_BASE_URL}${BASE_API_URL}/course_id/${courseId}`,
-  );
-  if (sequenceId) {
-    url.searchParams.append('content_id', sequenceId);
+  let data;
+  if (!getConfig().EXAMS_BASE_URL) {
+    const url = new URL(
+      `${getConfig().LMS_BASE_URL}${BASE_API_URL}/course_id/${courseId}`,
+    );
+    if (sequenceId) {
+      url.searchParams.append('content_id', sequenceId);
+    }
+    url.searchParams.append('is_learning_mfe', true);
+    const urlResponse = await getAuthenticatedHttpClient().get(url.href);
+    data = urlResponse.data;
+  } else {
+    const examUrl = new URL(`${getConfig().EXAMS_BASE_URL}/exam/attempt/course_id/${courseId}/content_id/${sequenceId}`);
```

Comment:
NIT: I personally would group these by the data type, e.g:

- examURL
- examResponse
- data = examResponse.data

- activeAttemptUrl
- activeAttemptResponse
- data.active_attempt = ...

This is very minor thing though

Reply:
I was grouping by order (i.e. create the URL for each, call each endpoint, then create the data response), but I can see how either are good